web.browser/src/containers/CreatePost/index.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import styles from './styles.css';
+import TextField from 'material-ui/TextField';
+import { Card } from 'material-ui/Card';
+import { Toolbar, ToolbarTitle } from 'material-ui/Toolbar';
+import FlatButton from 'material-ui/FlatButton';
+import { connect } from 'react-redux';
+import SelectField from 'material-ui/SelectField';
+import Gandalf from '../../lib/gandalf-validator/gandalf';
+import MenuItem from 'material-ui/MenuItem';
+import { fetchCategories } from './../../actions/CreatePostActions';
+
+class createPost extends Gandalf {
+
+  componentWillMount() {
+    this.props.fetchCategories();
+  }
+
+  constructor() {
+    const fields = {
+      title: {
+        component: TextField,
+        validators: ['required'],
+        errorPropName: 'errorText',
+        props: {
+          hintText: 'Title',
+          style: { width: 'calc(100% - 20px)' },
+        },
+      },
+      description: {
+        component: TextField,
+        validators: ['required'],
+        errorPropName: 'errorText',
+        props: {
+          hintText: 'Description',
+          style: { width: 'calc(100% - 20px)' },
+        },
+        debounce: 300,
+      },
+      link: {
+        component: TextField,
+        validators: ['required'],
+        errorPropName: 'errorText',
+        props: {
+          hintText: 'Link',
+          style: { width: 'calc(100% - 20px)' },
+        },
+      },
+      tags: {
+        component: TextField,
+        validators: ['required'],
+        errorPropName: 'errorText',
+        props: {
+          hintText: 'Tags',
+          style: { width: 'calc(100% - 20px)' },
```

Comment:
You could store this `style` string in a variable to prevent all the copy/pasting.